### ISSUE TITLE
Search results are a bit confusing or don't return anything and other issues.

### ISSUE THREAD

**Issue by MR4Y** [mps-youtube/mps-youtube#715]
I'm sort of measuring this against actual youtube and...the search results are very different from one to the other, that when the program actually returns any result.

I presume it only looks for music videos, as it returns search results that have nothing to do with the search terms I typed?

Also, what do I have to set to watch video without having to download it?

**Comment by ritiek** [comment]
> I presume it only looks for music videos, as it returns search results that have nothing to do with the search terms I typed?

Yes, this tool looks for only music videos by default. You can turn off this setting by `set search_music false`.

> Also, what do I have to set to watch video without having to download it?

What player are you using? I use mpv-player and it doesn't seem to download anything for me.

**Comment by MR4Y** [comment]
I have both mpv and vlc. Both only play music without showing video. Also, I don't want to download any videos I watch. My disk space is limited and having to keep track of files and deleting them afterwards would be a hassle.

**Comment by Razesdark** [comment]
You have to `set show_video true` to see video.

mpsyt stores the current file and might precache the next file in your playlist, but it should delete the file when it no longer needs it.

What OS and mpsyt version are you using?

**Comment by MR4Y** [comment]
Arch Linux. mpsyt version 0.2.7.1

For now, I set the video download folder to be /tmp

**Comment by ids1024** [comment]
Playing a video in mpsyt shouldn't download it to disk; the streaming url is passed to mpv/mplayer/etc.

@MR4Y do you still have any issues after setting `search_music` and `show_video`?

**Comment by MR4Y** [comment]
@ids1024 It works fine once these two were set.
